src/Form/PreferencesForm.php

```diff
@@ -136,19 +136,22 @@ public function buildForm(
     // Build form according to retrieved configuration:
     // Add contact fields.
     foreach ($profile->contact_fields as $contact_field_name => $contact_field) {
-      if ($contact_field['active']) {
-        $form[$contact_field_name] = [
+      if (isset($contact_field['active']) && $contact_field['active']) {
+        $form['contact-fields'][$contact_field_name] = [
           '#type' => Utils::getContactFieldType($contact_field),
           '#title' => $contact_field['label'],
           '#description' => $contact_field['description'],
           '#default_value' => $subscription['contact'][$contact_field_name],
           '#required' => !empty($contact_field['required']),
           '#disabled' => TRUE,
+          '#weight' => isset($contact_field['weight'])
+                    && filter_var($contact_field['weight'], FILTER_VALIDATE_INT) !== false
```

Comment:
`s/false/FALSE` according to [Drupal Coding Standards](https://www.drupal.org/docs/develop/standards/php/php-coding-standards#s-constants).

Also, please check for other occurrences.